src/utils/currency.ts

```
@@ -0,0 +1,26 @@
+export const formatCurrency = (amount: number | string): string => {
+  const currencyCode = import.meta.env.VITE_CURRENCY_CODE || 'IDR';
+  const currencySymbol = import.meta.env.VITE_CURRENCY_SYMBOL || 'Rp';
+
+  const numAmount = typeof amount === 'string' ? parseFloat(amount) : amount;
+
+  if (isNaN(numAmount) || !isFinite(numAmount)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)